v2/fleet-local/control/flight-director@.service

```diff
@@ -2,16 +2,18 @@
 Description=Flight Director @ %i
 # it's implied that chronos & marathon require mesos
 # look here: https://github.com/behance/mesos-systemd/tree/master/v2/fleet
-After=docker.service chronos.service marathon.service
+After=docker.service chronos@%i.service marathon@%i.service
```

Comment:
Yeah. I was testing against a fresh control tier though and chronos was never started yet FD did. So I'm pretty sure that it's functioning correctly.

I did consciously leave this here though, as FD should not be started until after ALL mesos frameworks start up. This way, in clusters where we *do* use chronos, systemd at least attempts to take its presence into account when starting the FD unit